boards/ti/lp_mspm0g3507/Kconfig.lp_mspm0g3507

```diff
@@ -2,4 +2,4 @@
 # SPDX-License-Identifier: Apache-2.0
 
 config BOARD_LP_MSPM0G3507
-	select SOC_MSPM0G3507
+	select SOC_MSPM0G3507
```

Comment:
don't know what is going on here but this is altering existing files for no reason?